omnibus/resources/agent/msi/cal/customaction.h

```diff
@@ -40,7 +40,7 @@ int uninstallServices(MSIHANDLE hInstall, CustomActionData& data);
 int verifyServices(MSIHANDLE hInstall, CustomActionData& data);
 
 //delfiles.cpp
-BOOL DeleteFilesInDirectory(const wchar_t* dirname, const wchar_t* ext);
+BOOL DeleteFilesInDirectory(const wchar_t* dirname, const wchar_t* ext, bool dirs = false);
```

Comment:
can you add some docs on this function, including some details on its behavior and a description of its parameters?